### PR TITLE
Feature/location api

### PR DESCRIPTION
**Pull request by mateomotriz**
the database is returning a null value :(

`  Future getLib() {
    Firestore.instance.runTransaction((Transaction tx) async {
      DocumentSnapshot snapshot = await tx.get(documentHM);
      if (snapshot.exists) {
        snapLat = await snapshot.data[bibLat].toString();
        snapLon = await snapshot.data[bibLon].toString();
      }
      return snapLat;
    });
  }`

### CODE REVIEW

**Comment by andreBrauni** [comment]
documentHM.get()
        .then((DocumentSnapshot document) {
      GeoPoint lib = document['point']
    });

File: lust/lib/models/bibs.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//To specify the bib: latitude and longitude
+
+class Library2 {
+  final double latitude;
+  final double longitude;
+
+  Library2(this.latitude, this.longitude);
+}
```
Comment (worldpotato) [inline lust/lib/models/bibs.dart]: can we delete that?

File: lust/pubspec.lock~getting coordinates from Firebase [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+  vector_math:
+    dependency: transitive
+    description:
+      name: vector_math
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "2.0.8"
+sdks:
+  dart: ">=2.1.0 <3.0.0"
+  flutter: ">=1.2.0 <2.0.0"
```
Comment (worldpotato) [inline lust/pubspec.lock~getting coordinates from Firebase]: pubspec.lock should not be in the repo

**Comment by worldpotato** [comment]
merged into #100

**Pull request closed without merging**